model/godleyTableWindow.cc

```diff
@@ -86,17 +86,19 @@ namespace minsky
       case 0:
         godleyIcon.table.insertRow(idx+1);
         break;
-      case 1:
-        godleyIcon.deleteRow(idx+1);
+      case 1:      
```

Comment:
It is better practice to have separate pull request for separate issues, as it make reviewing and so on easier. But OK, lets not worry about that now.

Reply:
Should I rather only submit one PR at a time? I ask because when I tried to create a second pull request (across forks) for ticket 1061, Github would only let me edit the one for ticket 1064 and not create an additional one.

Reply:
I fell into this trap the first time I did a pull request. Each pull request has its own branch. Committing to that branch updates the pull request. So instead of pull requesting of the one branch (fixes), you would have separate branches bugfix-1061, bugfix-1064 (these are just suggested names, you can name them how you like) based from highperformancecoder:master, and cherry pick the changesets you want into those branches. Instead of cherry picking, you can just work in the bugfix/feature branches, then have a "develop" branch which is a merge of the pull request branches for your own testing/usage purposes. Either way works.

But yes - you can have multiple pull requests open concurrently, but each needs its own branch. Please keep the pull requests to a single issue in the future. I don't mind making the exception for this one, but in this case #1061 is fixed completely, but #1064 still has a bit of work, so we could have merged one PR already :).

Reply:
Thank you for clearing that up, it can be confusing. I will have separate branches for bugfixes and feature additions from now on.